administrator/modules/mod_login/src/Helper/LoginHelper.php

```diff
@@ -25,14 +26,18 @@
  *
  * @since  1.6
  */
-abstract class LoginHelper
+class LoginHelper
 {
     /**
      * Get an HTML select list of the available languages.
      *
+     * @param   CMSApplicationInterface  $app  The application
+     *
      * @return  string
+     *
+     * @since   __DEPLOY_VERSION__
      */
-    public static function getLanguageList()
+    public function getLanguages(CMSApplicationInterface $app)
```

Comment:
```suggestion
    public function getLanguages(Registry $moduleParams, SiteApplication $app)
```
I wonder why you deviate from the previous converted modules? See, among others eg:
https://github.com/joomla/joomla-cms/blob/fba52f5c7066ff6be85de8b8a46e471379802fc8/modules/mod_articles_archive/src/Helper/ArticlesArchiveHelper.php#L47

Reply:
So may I mark conversation as resolved?

Reply:
yes